src/main/resources/assets/styles/main.css

```diff
@@ -85,4 +85,22 @@
         font-style: italic;
         margin-top: 10px;
     }
+
+    table {
+        border-collapse: collapse;
+        text-align: left;
+        min-width: 100%; /* mitigates aggressive word-wrap on caption and cells */
+    }
+    caption {
+        font-weight: bold;
+        text-align: left;
+    }
+    th, td {
+        border: 1px solid lightgray;
+        padding: 4px 7px;
+    }
+
+    details summary {
+        cursor: pointer;
+    }
```

Comment:
These are too generic. Widget resources are not isolated, so these styles will affect all the elements in the Content Studio's DOM. 

We need to either use custom style names prefixed with "widget-booster-", or scope generic styles under custom styles (f.ex. "`#widget-booster table`").

Reply:
Flattened the CSS declarations